### PR TITLE
Feat set 1

### PR DESCRIPTION
**Pull request by krishnakant01**
feat add: alert and avatar

### CODE REVIEW

**Comment by krishnakant01** [comment]
Live demo: https://culer-ui.netlify.app

**Review by tushartiwari7 — COMMENTED** [review]
Code looks fine
but please deploy on netlify/vercel to get PR preview domains.

**Review by tushartiwari7 — COMMENTED** [review]
Try making the sidebar and header of the app fixed and just the docs area scroll.

File: components/navigation-bar.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.nav-tagline {
+    font-weight: 500;
+    font-size: 1rem;
+    color: #1f1f1f;
+}
+
+.nav-links {
+    
+}
```
Comment (harshitadaswani) [inline components/navigation-bar.css:35]: It is not a good practice to define empty classes. You can either comment it out or remove it.

File: style.css [hunk 2]
```diff
@@ -117,35 +56,92 @@ body {
...
+    text-decoration: none;
+    color: var(--color-darker-gray);
+}
+.main-container{
+    grid-area: main;
+    box-shadow: 0 0 1px 0px rgb(0 0 0 / 25%);
+    padding: 0rem 2rem 2rem 2rem;
+}
+
+.component-container{}
```
Comment (harshitadaswani) [inline style.css:133]: Here also, you have defined empty class.

**Comment by krishnakant01** [comment]
> Try making the sidebar and header of the app fixed and just the docs area scroll.

yes, will fix that in the next merge. thanks.

**Review by harshitadaswani — APPROVED** [review]
Just a suggestion: You can make the sidebar either fixed or sticky to have better user accessibility.
Apart from that, the code looks good.

File: style.css (hunk 2 above)
Comment (krishnakant01) [reply style.css:133]: yes, those were for my reference. I'll delete it later. Thanks.

**Comment by ozairkhan2810** [comment]
Looks good apart from that aside said by Tushar.

**Pull request merged**